packages/reports/src/Containers/open-positions.jsx

```diff
@@ -309,10 +310,28 @@ const OpenPositions = ({
     server_time,
     ...props
 }) => {
-    const [active_index, setActiveIndex] = React.useState(is_multiplier ? 1 : 0);
     // Tabs should be visible only when there is at least one active multiplier contract
     const [has_multiplier_contract, setMultiplierContract] = React.useState(false);
+    const [contract_type_value, setContractTypeValue] = React.useState('Options');
     const previous_active_positions = usePrevious(active_positions);
+    const contract_types = [
+        {
```

Comment:
We should only display the available contract options, if client does not have any open positions then there is no point of having options.

Reply:
Can make dynamic dropdown but at the current production implementation "Options" and "Multiplier" tabs are there regardless of available open position and placeholder component is in the open positions page.

There's no clear cut design on whether they want to remove option without available open position or keep the option and continue using  a placeholder component. 

Should I consult with the design team on how they want the implementation to look like? Let me know how you want to proceed.

Reply:
Yes please ask for confirmation from design team for the best UX

Reply:
Design team will keep static dropdown menu regardless of (un)available open position for contract type.

https://deriv-group.slack.com/archives/C0YTBCXM1/p1674715536419229?thread_ts=1674713177.455879&cid=C0YTBCXM1